shared/profile/index.desktop.js

```diff
@@ -249,6 +249,8 @@ class ProfileRender extends PureComponent<Props, State> {
       }
     }
 
+    // TODO/songgao: is it intended that this prop is still here? The prop is
+    // not provided in the container at all.
```

Comment:
I think this is the last major use of `constants/types/folders.js`. It seems to intend to show a list of folders by the user, but is not visible in the app now. Do we still want it shown?